client/src/components/Login/index.jsx

```diff
@@ -6,8 +6,31 @@ export default class extends Component {
   state = {
     email: '',
     password: '',
+    msg:''
   };
+  handelChange = ({ target: { name, value } }) => {
+    if(name ==="email"&& value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i))
+    this.setState({ [name]: value })
+    else this.setState({[name]:value})
+  }
+  handelSubmit = (e) => {
+    this.setState({msg:''})
+    e.preventDefault();
+    const { email, password } = this.state
+    if(email && password){
+    fetch('/api/login', {
+      credentials: 'same-origin',
+      method: 'POST',
+      headers: {'Content-Type': 'application/json' },
+      body: JSON.stringify({email,password})
+    }).then(res =>  res.json())
+    .then(res => {
+      if(res !=='done') 
```

Comment:
we should not check for string. I prefer that we send a response object.
to determine the message we show the user, I suggest we check for status codes rather than messages. In many cases, we might not even need a response object, just a response status.

Reply:
ok